test/vast_parser.js

```diff
@@ -882,6 +882,12 @@ describe('VASTParser', function() {
       it('should have carried sequence over from wrapper', () => {
         this.response.ads[0].sequence.should.eql('1');
       });
+
```

Comment:
please add tests in the new stack we have (jest) under spec folder. You should rebase from 3.0-version branch beforehand though because vast_parser.js file was changed a lot and this pr is behind those changes.